### PR TITLE
Move Websocket commands doc into docs directory

### PR DESCRIPTION
**Pull request by lboue**
- To shorten the main page, move Websocket commands doc into docs directory
- Add 'Read an attribute' in README
- Add 'Write an attribute' to README

### CODE REVIEW

**Comment by agners** [comment]
Maybe it would be better if we have the WS protocol description in a separate docs folder? :thinking:

**Comment by lboue** [comment]
> Maybe it would be better if we have the WS protocol description in a separate docs folder? 🤔

I tend to agree with that.

**Comment by lboue** [comment]
Is it OK for you this way?

File: README.md [hunk 1]
```diff
@@ -342,3 +172,7 @@ There is also a Python client library hosted in this repository (used by Home As
 The client library has a dependency on the chip/matter clusters package which contains all (Cluster) models and this package is os/platform independent. The server library depends on the Matter Core SDK (still named CHIP) which is architecture and OS specific. We build (and publish) wheels for Linux (amd64 and aarch64) to pypi but for other platforms (like Macos) you will need to build those wheels yourself using the exact same version of the SDK as we use for the clusters package. Take a look at our build script for directions: https://github.com/home-assistant-libs/chip-wheels/blob/main/.github/workflows/build.yaml
 
 To only install the client part: `pip install python-matter-server`
+
+### Websocket commands
+
+[Websocket documentation](/docs/README.md)
```
Comment (marcelveldt) [inline README.md:178]:
```suggestion
[Websocket documentation](/docs/websockets_api.md)
```

Comment (lboue) [reply README.md]: Fixed

**Comment by marcelveldt** [comment]
Thanks!

**Pull request merged**